src/ar_storage.erl

```diff
@@ -325,7 +327,7 @@ write_tx(Tx) ->
 		Name = lists:flatten(
 			io_lib:format(
 				"~s/~s.json",
-				[?TX_DIR, ar_util:encode(Tx#tx.id)]
+				[ar_meta_db:get(data_dir) ++ "/" ++ ?TX_DIR, ar_util:encode(Tx#tx.id)]
```

Comment:
I would like to put the whole `lists:flatten/1` call in a function that takes a transaction as argument.

Reply:
✅